airflow-core/src/airflow/cli/commands/plugins/react_plugin_template/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "{{PROJECT_NAME}}",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "main": "./dist/main.js",
+  "module": "./dist/main.js",
+  "types": "./dist/main.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/main.js",
+      "types": "./dist/main.d.ts"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "dev": "vite --port 5173 --strictPort",
+    "build": "vite build",
+    "build:types": "tsc --p tsconfig.lib.json",
+    "build:lib": "vite build",
+    "lint": "eslint --quiet && tsc --p tsconfig.app.json",
+    "lint:fix": "eslint --fix && tsc --p tsconfig.app.json",
+    "format": "pnpm prettier --write .",
+    "preview": "vite preview",
+    "test": "vitest run",
+    "coverage": "vitest run --coverage"
+  },
+  "dependencies": {
+    "@chakra-ui/anatomy": "^2.3.4",
+    "@chakra-ui/react": "^3.20.0",
+    "@emotion/react": "^11.14.0",
+    "@tanstack/react-query": "^5.75.1",
+    "@tanstack/react-table": "^8.21.3",
+    "@tanstack/react-virtual": "^3.13.8",
+    "axios": "^1.8.4",
+    "chakra-react-select": "6.1.0",
+    "dayjs": "^1.11.13",
+    "i18next": "^25.1.2",
+    "i18next-browser-languagedetector": "^8.1.0",
+    "i18next-http-backend": "^3.0.2",
+    "next-themes": "^0.3.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-hook-form": "^7.56.1",
+    "react-i18next": "^15.5.1",
+    "react-icons": "^5.5.0",
+    "react-router-dom": "^6.30.0",
+    "use-debounce": "^10.0.4",
+    "usehooks-ts": "^3.1.1",
+    "zustand": "^5.0.4"
```

Comment:
We should probably simplify the dependencies here to just what is necessary to render a chakra app that can make API calls.

So lets drop zustand, react-table, react-virtual, etc